src/apis/posts.js

```diff
@@ -0,0 +1,55 @@
+import { get, del, patch } from "../libs/api/apiWrapper";
+
+import APIs from "../helpers/apiRoutes";
+
+/**
+ * Get posts by role id
+ * @param {number} id - The role ID for which to retrieve posts.
+ * @returns {Promise} A Promise that resolves to the post data or rejects with an error.
+ */
+const getPosts = async () => {
+  try {
+    const response = await get(`${APIs.POSTS.GET_POSTS}`, {
+      detachRoleToken: false,
```

Comment:
No need to pass detachRoleToken and detachAccessToken keys

Reply:
Fixed